packages/adapters-library/src/scripts/helpers.ts

```diff
@@ -42,15 +42,15 @@ export class Helpers {
     public readonly allJsonRpcProviders: Record<Chain, CustomJsonRpcProvider>,
   ) {}
 
-  async getProtocolTokenByAddress<
```

Comment:
Any issue removing the `async` here? It shouldn't be asynchronous and it's adding overhead.